lib/matplotlib/cbook.py

```diff
@@ -566,6 +566,18 @@ def process(self, s, *args, **kwargs):
                     self._remove_proxy(proxy)
 
 
+class EventEmitter(object):
+    def __init__(self):
+        super(EventEmitter, self).__init__()
```

Comment:
Thanks, I shall get onto it in due course, and perhaps do a bit of a clearout at the same time, as I got a bit confused last time I looked as some of it looks out of date.

Doing a search for `super` in matplotlib https://github.com/matplotlib/matplotlib/search?utf8=%E2%9C%93&q=super shows we have gone a long way past the **if** stage, so I think we do need to embrace it.

The ``against`` article you linked to (I read it 2 days ago) seems to either set up straw-man arguments against super, as in they don't provide any sort of rebuttal against the ways you should use it, i.e. that explained in the pro-article; or give arguments against that also apply if you don't use super.